Components/ArcadiaBehaviour.cs

```diff
@@ -108,12 +114,19 @@ public virtual void Awake()
 						var nameSym = Symbol.intern(sym.Name);
 						var nsSym = Symbol.intern(sym.Namespace);
 						requireFn.invoke(nsSym);
-						var v = Namespace.find(nsSym).FindInternedVar(nameSym);
-						fnList.Add(v);
+						try
+						{
+							var v = Namespace.find(nsSym).FindInternedVar(nameSym);
+							fnList.Add(v);
+						} catch (System.Exception e)
+						{
+							fnList.Add(RT.var(sym.Namespace, sym.Name));
+							Debug.LogError(new System.Exception("ArcadiaBehaviour: Can't find #'"+ sym.Namespace+"/"+sym.Name));
```

Comment:
Maybe a message along the lines of "Exception thrown while attaching #'var/name to *HookName*"? You can get the name of the current class with `this.GetType().Name`.

We can also include the original exception as an inner exception of the new exception we're throwing here, using [the two-arity constructor](https://msdn.microsoft.com/en-us/library/804f22sf(v=vs.110).aspx). I am not sure if Unity will print that out, though, but that's the best way to preserve error information.